packages/contracts-bedrock/src/governance/GovernanceTokenInterop.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { ERC20Votes } from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { IGovernanceDelegation } from "src/governance/IGovernanceDelegation.sol";
+import { GovernanceToken } from "src/governance/GovernanceToken.sol";
+
+contract GovernanceTokenInterop is GovernanceToken {
+    /// @notice Returns the checkpoint for a given account at a given position.
+    /// @param _account The account to get the checkpoints for.
+    /// @param _pos     The psition to get the checkpoints at.
+    /// @return         The checkpoint at the given position.
+    function checkpoints(address _account, uint32 _pos) public view override(ERC20Votes) returns (Checkpoint memory) {
+        if (_migrated(_account)) {
+            return IGovernanceDelegation(Predeploys.GOVERNANCE_DELEGATION).checkpoints(_account, _pos);
```

Comment:
Honestly the change to using L1 block number is probably breaking enough that this isn't even worth doing anymore. I think we can save that discussion for the interop spec though.

Reply:
agree, just updated govdelegation to only migrate latest but migration can eventually be removed in interop version